packages/legacy-workbench/src/js/angular/guides/tour-lib-services/shepherd.service.js

```diff
@@ -212,7 +215,11 @@ function ShepherdService($translate, LocalStorageAdapter, LSKeys, $interpolate,
             confirmCancel: this._confirmGuideCancel,
             defaultStepOptions: {
                 classes: 'shadow-md bg-purple-dark',
-                scrollTo: true
+                scrollTo: {
```

Comment:
I know that we can't use the typescript types in the legacy, but still I don't like having these 3 essentially duplicated types for these constants. I'd used just constant strings instead

Reply:
Done